src/Lykke.Service.BlockchainSettings/Modules/CacheModule.cs

```diff
@@ -31,27 +33,33 @@ public CacheModule(IReloadingManager<CacheSettings> settings, ILog log)
 
         protected override void Load(ContainerBuilder builder)
         {
-            //_services.AddDistributedRedisCache(options =>
-            //{
-            //    options.Configuration = _settings.CurrentValue.RedisConfiguration;
-            //    options.InstanceName = _settings.CurrentValue.InstanceName;
-            //});
+            var settingsCurrentValue = _settings.CurrentValue;
+            IDistributedCache cache = null;
 
-            var redis = new RedisCache(new RedisCacheOptions
+            if (settingsCurrentValue == null ||
+                string.IsNullOrEmpty(settingsCurrentValue.InstanceName) ||
+                string.IsNullOrEmpty(settingsCurrentValue.RedisConfiguration))
             {
-                Configuration = _settings.CurrentValue.RedisConfiguration,
-                InstanceName = _settings.CurrentValue.InstanceName
-            });
+                var inMemoryCacheOptions = Options.Create(new MemoryDistributedCacheOptions()
+                {
+                });
 
-            builder.RegisterInstance(redis)
+                cache = new MemoryDistributedCache(inMemoryCacheOptions);
+            }
+            else
+            {
+                cache = new RedisCache(new RedisCacheOptions
+                {
+                    Configuration = _settings.CurrentValue.RedisConfiguration,
+                    InstanceName = _settings.CurrentValue.InstanceName
```

Comment:
Please add root key "BlockchainSettings" and handle null InstanceName as here https://github.com/LykkeCity/LykkePublicAPI/blob/master/src/LykkePublicAPI/Modules/ApiModule.cs#L141